DataCenterBridging.psm1

```diff
@@ -1276,7 +1340,9 @@ function Test-FabricInfo {
         after the function call did not have an LLDP 10041 packet in the event log.
     #>
     $global:IndexesMissingEvents = $Null
-    $event = Get-LLDPEvents -RemainingIndexes $remainingIndexes

```

Comment:
Good call, $Event is a keyword. I need to stop using that.

Reply:
This is why I use VSCode for all my PowerShell stuff. It adds slip-ups like this in the Problems tab you when you do something wrong or aren't playing nice with the PowerShell best practices. It can be annoying at times, but it keeps the script clean and free of the bugs that should be obvious but never are.